src/test/java/th/ac/kmitl/soa/group2/models/DefinedTradeContactModelTest.java

```diff
@@ -0,0 +1,27 @@
+package th.ac.kmitl.soa.group2.models;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static th.ac.kmitl.soa.group2.utils.XmlFormatter.serialize;
```

Comment:
Let’s praise for this line lol. Because I renamed ‘XmlFormatter’ to ‘binders.Xml’

Reply:
55555